src/main/java/com/tabnine/statusBar/StatusBarPromotionWidget.java

```diff
@@ -89,10 +89,11 @@ public String getTooltipText() {
             if (!e.isPopupTrigger() && MouseEvent.BUTTON1 == e.getButton()) {
                 binaryRequestFacade.executeRequest(new StatusBarPromotionActionRequest(
                         component.getId(), component.getText(), component.getActions()));
-                clearMessage();
-                if (component.getActions().contains(StaticConfig.OPEN_HUB_ACTION)) {
+                List<Object> actions = component.getActions();
+                if(actions != null && actions.stream().anyMatch(a -> a instanceof String && a.toString().equals(StaticConfig.OPEN_HUB_ACTION))){
```

Comment:
I think you can just do OPEN_HUB_ACTION.equals(a) and it will check the type (if a is a String, there's no need to call toString on it).